### PR TITLE
Improve project structure

### PR DESCRIPTION
**Pull request by rrye**

### CODE REVIEW

File: reposilite-frontend/src/views/landing/home/style.styl [hunk 1]
```diff
@@ -0,0 +1,32 @@
+@import url('https://fonts.googleapis.com/css?family=Titillium+Web')
+@import url('https://fonts.googleapis.com/css?family=Raleway')
+@import url('https://fonts.googleapis.com/css?family=Rubik')
+
+#home-wrapper
+    background-color #2c3e50
+
+h1
+    color #e6e6e6;
```
Comment (dzikoysk) [inline reposilite-frontend/src/views/landing/home/style.styl]: :O

File: reposilite-frontend/src/components/search/component.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+                    ]
+                }
+            ]
+        }
+    },
+    methods: {
+        onInputChange(text) {
+            if (!text) {
+                this.filteredOptions = []
+                return;
```
Comment (dzikoysk) [inline reposilite-frontend/src/components/search/component.js]: :O

File: reposilite-frontend/src/components/search/component.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+            }
+
+            const filteredData = this.suggestions[0].data.filter(option => {
+                return option.name.toLowerCase().indexOf(text.toLowerCase()) > -1
+            });
```
Comment (dzikoysk) [inline reposilite-frontend/src/components/search/component.js]: :O

File: reposilite-frontend/src/components/search/component.js [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+            this.filteredOptions = [{ data: filteredData }];
```
Comment (dzikoysk) [inline reposilite-frontend/src/components/search/component.js]: :O

File: reposilite-frontend/src/components/search/component.js [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+        },
+        renderSuggestion(suggestion) {
+            const character = suggestion.item;
```
Comment (dzikoysk) [inline reposilite-frontend/src/components/search/component.js]: :O

File: reposilite-frontend/src/components/search/component.js [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+                            width: "25px",
+                            height: "25px",
+                            borderRadius: "15px",
+                            marginRight: "10px"
+                        }}
+                        src={ character.avatar }
+                    />
+                    <span style={{ color: "navyblue" }}>{character.name}</span>
+                </div>
+            );
```
Comment (dzikoysk) [inline reposilite-frontend/src/components/search/component.js]: :O

**Comment by wvffle** [comment]
@dzikoysk next time you approve some frontend changes, check for new lines >:(

**Pull request merged**